### PR TITLE
Task3, 5 - add `-c` flag to display highest & lowest temperatures of each day in a month

### PR DESCRIPTION
**Pull request by abdul-moize**
This PR implements Task 3 and 5 of weatherman assignment. Task 3 is basically printing extreme temperatures of each day of a month. Extreme temperatures are printed as colored '+' signs(blue for lowest, red for highest). Task 5 is the same as Task 3 but it prints everything on the same line.

To test this PR. follow these steps:
1. pull this branch
2. run this command `pip install -r requirements.txt`
3. run this command afterwards, `weatherman.py -c 2006/6`
This will print the extreme temperatures for the 6th month of 2006.
Test this PR with different values and please let me know if you come across any issues.

### CODE REVIEW

File: modules/charts_month.py [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+from constants import months_list
+from modules.utils import (
+    get_date,
+    get_highest_temperature,
+    get_lowest_temperature,
+    read_data,
+)
+
+
+def get_all_extremes(year_month, path):
```
Comment (arslanashraf7) [inline modules/charts_month.py]: We should separate the year and month into 2 params as we discussed in Task2.

Comment (arslanashraf7) [inline modules/charts_month.py]: The default value for path? Could it be from constants?

**Review by arslanashraf7 — CHANGES_REQUESTED** [review]
Some minor comments as i'll majorly be checking it as a whole PR for the complete weatherman task, The PR that you will create from master to main.

Additionally, Some minor suggestions on this one, and also please add the  PyCache .pyc files to gitignore.

File: modules/charts_month.py (hunk 1 above)
Comment (abdul-moize) [reply modules/charts_month.py]: done

Comment (abdul-moize) [reply modules/charts_month.py]: done

**Review by arslanashraf7 — APPROVED** [review]
Looks good 👍 , Let's squash and merge this into master.

**Edit:***
Sorry, I didn't notice but I can still see Pycache files in the commits.

As a more generic process or branching, let's make a PR from master to main branch once this PR is merged into master.

**Comment by abdul-moize** [comment]
> Looks good +1 , Let's squash and merge this into master.
> 
> **Edit:***
> Sorry, I didn't notice but I can still see Pycache files in the commits.
> 
> As a more generic process or branching, let's make a PR from master to main branch once this PR is merged into master.

yes they got in through some previous commit during rebase. i have removed them now

**Pull request merged**